package.json

```diff
@@ -47,6 +50,6 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^3.8.1"
+    "cypress": "^4.5.0"
```

Comment:
I don't think this was needed, but thanks! 😬

Reply:
Not sure how this happened but updated. 👍🏽